_projects/konnektor.md

```diff
@@ -0,0 +1,9 @@
+---
+name: konnektor
+repo: OpenFreeEnergy/konnektor
+docs: https://konnektor.readthedocs.org
+conda-install: konnektor
+role: developers
+description: >-
+  [Konnektor](https://pubs.acs.org/doi/10.1021/acs.jcim.4c01710) is a package for planning, modifying, and analyzing free energy calculation networks.
```

Comment:
```suggestion
  [Konnektor](https://pubs.acs.org/doi/10.1021/acs.jcim.4c01710) is a package for planning, modifying, and analyzing free energy transformation networks.
```
Maybe a bit pendantic, but suggesting transformation over calculation because you can have calculation that eats up multiple transformation (RE-EDS and ladybugs are examples here).